### PR TITLE
Bug in large binary update

### LINKED ISSUE #167: Large binary transactions export issue
### Description

The Large Binary File update creation workflow does not succeed: The system does not recognize the transaction

### Steps to reproduce

Try to export a large binary transaction after creation

### Additional context

_No response_

### Hedera network

other

### Version

All versions

### Operating system

macOS

### PR DESCRIPTION
**Pull request by DSMatusevich**
**Description**:
Fixed a bug where the large binary update files would not be created.

**Related issue(s)**:

Fixes #167 

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: tools-ui/src/test/java/com/hedera/hashgraph/client/ui/CreatePaneControllerTest.java [hunk 1]
```diff
@@ -950,6 +956,122 @@ public void createUpdateAccount_Test() throws HederaClientException {
 		assertTrue(comment.has("Timestamp"));
 	}
 
+	@Test
+	public void createLargeBinaryUpdate_test() throws HederaClientException, IOException {
```
Comment (qnswirlds) [inline tools-ui/src/test/java/com/hedera/hashgraph/client/ui/CreatePaneControllerTest.java:960]: nit: This test is long and hard to follow. Should consider breaking it up into smaller helper functions, especially to show clearly the test setup (given, when) and the expectation (then). Also, many of the var here could also be `final`.

**Review by qnswirlds — APPROVED** [review]
Overall looks ok to me.

File: tools-ui/src/test/java/com/hedera/hashgraph/client/ui/CreatePaneControllerTest.java (hunk 1 above)
Comment (DSMatusevich) [reply tools-ui/src/test/java/com/hedera/hashgraph/client/ui/CreatePaneControllerTest.java:960]: Thanks I will break it up
